website/src/pages/log-streaming/content-overview/index.mdx

```diff
@@ -77,7 +77,7 @@ Lets get started by following the steps below:
 
 </InlineNotification>
 
-If your Event Streaming has been whitelisted, you should see a screen similar to the one shown below. 
+If your Log Streaming has been whitelisted, you should see a screen similar to the one shown below. 
```

Comment:
Remove this and the notification section before it. Whitelisting is no longer required for an account to use the Log Streaming feature.